app/controllers/staff/adjustments_controller.rb

```diff
@@ -0,0 +1,71 @@
+class Staff::AdjustmentsController < Staff::ActivitiesController
+  include Secured
+
+  def new
+    @activity = activity
+    @adjustment = AdjustmentForm.new
+    @reports = Report.for_activity(@activity)
+    @adjustment.activity = @activity
+
+    authorize(@adjustment, policy_class: AdjustmentPolicy)
+  end
+
+  def create
+    @adjustment = AdjustmentForm.new(params[:adjustment_form])
+    return show_errors unless @adjustment.valid?
+
+    @activity = activity
+    result = create_adjustment
+    @adjustment = authorize(result.object)
```

Comment:
maybe, I'm a bit sketchy on Pundit. I'll take another look at this...

Reply:
yes, very good point. Not much point authorising _after_ the Adjustment is created!

I got rather confused with my authorisations, I think because of the "form object". A key learning has been the syntax for telling pundit which policy to use for an arbitrary object:

```
authorize(@crab_apple, policy_class: ApplePolicy)
```